src/ck-ldb/TreeLevel.h

```diff
@@ -1065,19 +1070,25 @@ class PELevel : public LevelLogic
     if (nobjs > 0) lbmgr->GetObjData(allLocalObjs.data());  // populate allLocalObjs
     myObjs.clear();
     LBRealType nonMigratableLoad = 0;
+    size_t minPosDimension = std::numeric_limits<size_t>::max();
```

Comment:
use a mixture of **size_t** and **unsigned int** for min/maxPosDimensions among this PR

Reply:
Should be resolved in 2306daf.